posthog/hogql/transforms/test/__snapshots__/test_in_cohort.ambr

```diff
@@ -31,8 +31,8 @@
   FROM events LEFT JOIN (
   SELECT person_static_cohort.person_id AS cohort_person_id, 1 AS matched, person_static_cohort.cohort_id AS cohort_id 
   FROM person_static_cohort 
-  WHERE and(equals(person_static_cohort.team_id, 420), in(person_static_cohort.cohort_id, [6]))) AS __in_cohort ON equals(__in_cohort.cohort_person_id, events.person_id) 
```

Comment:
this for example was flapping pretty frequently on cohort id on unrelated changes for me